Aptoide/aptoide-client-v8
# [ASV-1147] Initial cobrand implementation

**What does this PR do?**

  Base implementation for the cobrand version - Including creation of the flavour and initial configuration.
  Android Home Aptoide shortcut was disabled in the cobrand version.
  Disabled adult content from cobrand - default is false.
  Disabled direct notifications from cobrand.
  
**Database changed?**

   No

**Where should the reviewer start?**

- [ ] build.gradle(app)
- [ ] gradle.properties

**How should this be manually tested?**

  Build vanilla, cobrand - check configuration (vername, vercode, package name, etc)

**What are the relevant tickets?**

  Tickets related to this pull-request: [ASV-1147](https://aptoide.atlassian.net/browse/ASV-1147)
[ASV-1171](https://aptoide.atlassian.net/browse/ASV-1171)
[ASV-1168](https://aptoide.atlassian.net/browse/ASV-1168)


**Code Review Checklist**
